test/window/frame.js

```diff
@@ -389,5 +389,31 @@ exports.tests = {
 
       test.done();
     });
+  },
+
+  'test frame references': function (test) {
+    var document = jsdom.jsdom("<!doctype html><iframe name='foo'></iframe>");
+    var window = document.defaultView;
+
+    test.strictEqual(window.length, 1, "frame should exist (.length)");
+    test.notEqual(window[0], undefined, "frame should exist (undefined check)");
+    test.strictEqual(window[0], window.foo, "index access and name access should return same reference");
+
+    test.done();
+  },
+
+  'remove frame': function (test) {
```

Comment:
Would like an additional test with removing from the middle of frames, e.g. with three frames remove frame 1